skipchain/struct.go

```diff
@@ -767,6 +771,35 @@ func (db *SkipBlockDB) StoreBlocks(blocks []*SkipBlock) ([]SkipBlockID, error) {
 						return fmt.Errorf("Tried to store unlinkable block: %+v", sb.SkipBlockFix)
 					}
 				}
+
+				// As we don't store the target height in the forward-link, there is no
+				// way to insure that it will be stored at the right height if the target
+				// block is not yet discovered like in a catch up scenario that will end
+				// up in this path. But we can at least check that the forward-link belongs
```

Comment:
I understand the first sentence until "that will end up in this path". Can you rephrase it? Or remove the final part it if it's not super important.

Reply:
I removed it, I guess it's more confusing than informative.